src/artemis/ispyb/store_in_ispyb.py

```diff
@@ -148,7 +148,6 @@ def _store_data_collection_group_table(self) -> int:
 
         params = self.mx_acquisition.get_data_collection_group_params()
         params["parentid"] = session_id
-        params["experimenttype"] = "mesh"
```

Comment:
Is there a reason for removing this? (although it has reminded me I need to update the 3d scan ispyb change to include changing this)

Reply:
Oops, typo, well spotted